flymake-languagetool.el

```diff
@@ -31,15 +31,14 @@
 
 ;;; Code:
 
+(require 'seq)
 (require 'url)
 (require 'json)
 (require 'flymake)
 
 ;; Dynamically bound.
 (defvar url-http-end-of-headers)
 
-(autoload #'seq-map "seq")
```

Comment:
Were there errors with this or just a style preference?

Reply:
style preferences.